spec/controllers/media_controller_spec.rb

```diff
@@ -79,4 +85,43 @@
       end
     end
   end
+
+  describe '#verify_token' do
+    let(:id) { 'ab123cd4567' }
+    let(:file_name) { 'interesting_video.mp4' }
+    let(:user_ip_addr) { '192.168.1.100' }
+    let(:token) { StacksMediaToken.new(id, file_name, user_ip_addr) }
+
+    # rubocop:disable Metrics/LineLength
```

Comment:
@jkeck:  yup, happy to pair/chat to come up with a way to shorten lines.  i couldn't think of anything offhand that didn't detract from readability or add what seemed like unnecessary temp vars.  i'm happy to go with whatever people want to do on line length.

@ndushay:  agreed, i also really dislike inline rubocop exceptions, but i saw an empty `.rubocop_todo.yml` and other parts of the code that had inline exceptions, so i figured i'd just stay consistent with the codebase.

anyway, i often find longer lines to be more readable than the things people do to avoid them, but again, not a cause i feel the need to fight for.